libs/langchain-cohere/src/chat_models.ts

```diff
@@ -135,7 +196,7 @@ export class ChatCohere<
 
   client: CohereClient;
 
-  model = "command";
+  model = "command-r-plus";
```

Comment:
This is technically breaking

Reply:
if we're going to break here, can we make model a required param

Reply:
Hey @billytrend-cohere We're making the model a required param for Embedding, and Reranker. Imo it is better to leave model as a non-required param for chat, since the implication of an unforeseen model bump isn't huge compared to embedding/rerank.

Reply:
Would still prefer this to be required (look at what happened with `gpt-4o`) but if you feel strongly I'm fine with keeping it.